dotcom-rendering/src/web/components/App.tsx

```diff
@@ -525,113 +437,6 @@ export const App = ({ CAPI, ophanRecord }: Props) => {
 					/>
 				</HydrateOnce>
 			))}
-			{qandaAtoms.map((qandaAtom) => (
-				<HydrateOnce rootId={qandaAtom.elementId}>
-					<QandaAtom
-						id={qandaAtom.id}
-						title={qandaAtom.title}
-						html={qandaAtom.html}
-						image={qandaAtom.img}
-						credit={qandaAtom.credit}
-						pillar={pillar}
-						likeHandler={componentEventHandler(
-							'QANDA_ATOM',
-							qandaAtom.id,
-							'LIKE',
-						)}
-						dislikeHandler={componentEventHandler(
-							'QANDA_ATOM',
-							qandaAtom.id,
-							'DISLIKE',
-						)}
-						expandCallback={componentEventHandler(
-							'QANDA_ATOM',
-							qandaAtom.id,
-							'EXPAND',
-						)}
-					/>
-				</HydrateOnce>
-			))}
```

Comment:
I see that for Qanda atoms (no idea what these are) we are no longer reporting ophan data. Is this perhaps something we can do in the wrapper to make sure we're not loosing functionality?

Reply:
The tracking requests still get sent. It's just I made these tracking props optional and then added a default inside the atoms themselves. 

https://github.com/guardian/atoms-rendering/pull/325

Reply:
👍 Thanks for clarifying!